slices/init-system-helpers.yaml

```diff
@@ -0,0 +1,25 @@
+package: init-system-helpers
+
+slices:
+  bins:
+    contents:
+      /usr/bin/deb-systemd-helper:
```

Comment:
those don't seem to be the right files for this slice. Check https://packages.ubuntu.com/noble/all/init-system-helpers/filelist

Reply:
see comment above: https://github.com/canonical/chisel-releases/pull/234/files#r1603091537

Reply:
Yeah, I was using to the output of `dpkg -c ` as a reference:

```shell
# dpkg -c procps_2%3a4.0.4-4ubuntu3_amd64.deb 
drwxr-xr-x root/root         0 2024-03-31 08:16 ./
drwxr-xr-x root/root         0 2024-03-31 08:16 ./etc/
drwxr-xr-x root/root         0 2024-03-31 08:16 ./etc/init.d/
-rwxr-xr-x root/root       959 2024-03-24 15:35 ./etc/init.d/procps
-rw-r--r-- root/root      2209 2024-03-24 15:35 ./etc/sysctl.conf
drwxr-xr-x root/root         0 2024-03-31 08:16 ./etc/sysctl.d/
-rw-r--r-- root/root        77 2024-03-31 08:16 ./etc/sysctl.d/10-console-messages.conf
-rw-r--r-- root/root       490 2024-03-31 08:16 ./etc/sysctl.d/10-ipv6-privacy.conf
-rw-r--r-- root/root      1229 2024-03-31 08:16 ./etc/sysctl.d/10-kernel-hardening.conf
-rw-r--r-- root/root      1184 2024-03-31 08:16 ./etc/sysctl.d/10-magic-sysrq.conf
-rw-r--r-- root/root       164 2024-03-31 08:16 ./etc/sysctl.d/10-map-count.conf
-rw-r--r-- root/root       158 2024-03-31 08:16 ./etc/sysctl.d/10-network-security.conf
-rw-r--r-- root/root      1292 2024-03-31 08:16 ./etc/sysctl.d/10-ptrace.conf
-rw-r--r-- root/root       506 2024-03-31 08:16 ./etc/sysctl.d/10-zeropage.conf
-rw-r--r-- root/root       798 2024-03-24 15:35 ./etc/sysctl.d/README.sysctl
...
```

Reply:
I'll update last comment, wrong package ...

Reply:
I may have mixed things up, will send a fixup

Reply:
This is the output for `dpkg -c` that I used as reference:

```shell
root@ef5890586639:/# dpkg -c init-system-helpers_1.66ubuntu1_all.deb 
drwxr-xr-x root/root         0 2023-12-06 07:50 ./
drwxr-xr-x root/root         0 2023-12-06 07:50 ./etc/
drwxr-xr-x root/root         0 2023-12-06 07:50 ./etc/rc0.d/
drwxr-xr-x root/root         0 2023-12-06 07:50 ./etc/rc1.d/
drwxr-xr-x root/root         0 2023-12-06 07:50 ./etc/rc2.d/
drwxr-xr-x root/root         0 2023-12-06 07:50 ./etc/rc3.d/
drwxr-xr-x root/root         0 2023-12-06 07:50 ./etc/rc4.d/
drwxr-xr-x root/root         0 2023-12-06 07:50 ./etc/rc5.d/
drwxr-xr-x root/root         0 2023-12-06 07:50 ./etc/rc6.d/
drwxr-xr-x root/root         0 2023-12-06 07:50 ./etc/rcS.d/
drwxr-xr-x root/root         0 2023-12-06 07:50 ./etc/systemd/
drwxr-xr-x root/root         0 2023-12-06 07:50 ./etc/systemd/system/
drwxr-xr-x root/root         0 2023-12-06 07:50 ./etc/systemd/user/
drwxr-xr-x root/root         0 2023-12-06 07:50 ./usr/
drwxr-xr-x root/root         0 2023-12-06 07:50 ./usr/bin/
-rwxr-xr-x root/root     24358 2023-12-06 07:46 ./usr/bin/deb-systemd-helper
-rwxr-xr-x root/root      7135 2023-12-06 07:47 ./usr/bin/deb-systemd-invoke
drwxr-xr-x root/root         0 2023-12-06 07:50 ./usr/sbin/
-rwxr-xr-x root/root     16517 2023-12-06 07:46 ./usr/sbin/invoke-rc.d
-rwxr-xr-x root/root      9104 2023-12-06 07:50 ./usr/sbin/service
-rwxr-xr-x root/root     18147 2023-12-06 07:47 ./usr/sbin/update-rc.d
drwxr-xr-x root/root         0 2023-12-06 07:50 ./usr/share/
drwxr-xr-x root/root         0 2023-12-06 07:50 ./usr/share/bug/
drwxr-xr-x root/root         0 2023-12-06 07:50 ./usr/share/bug/init-system-helpers/
-rw-r--r-- root/root        24 2023-12-06 07:46 ./usr/share/bug/init-system-helpers/control
drwxr-xr-x root/root         0 2023-12-06 07:50 ./usr/share/doc/
drwxr-xr-x root/root         0 2023-12-06 07:50 ./usr/share/doc/init-system-helpers/
-rw-r--r-- root/root      2365 2023-12-06 07:46 ./usr/share/doc/init-system-helpers/README.invoke-rc.d.gz
-rw-r--r-- root/root      1835 2023-12-06 07:46 ./usr/share/doc/init-system-helpers/README.policy-rc.d.gz
-rw-r--r-- root/root      3310 2023-12-06 07:50 ./usr/share/doc/init-system-helpers/changelog.gz
-rw-r--r-- root/root      3133 2023-12-06 07:46 ./usr/share/doc/init-system-helpers/copyright
drwxr-xr-x root/root         0 2023-12-06 07:50 ./usr/share/man/
drwxr-xr-x root/root         0 2023-12-06 07:50 ./usr/share/man/man1/
-rw-r--r-- root/root      2076 2023-12-06 07:50 ./usr/share/man/man1/deb-systemd-helper.1p.gz
-rw-r--r-- root/root      1456 2023-12-06 07:50 ./usr/share/man/man1/deb-systemd-invoke.1p.gz
drwxr-xr-x root/root         0 2023-12-06 07:50 ./usr/share/man/man8/
-rw-r--r-- root/root      2781 2023-12-06 07:50 ./usr/share/man/man8/invoke-rc.d.8.gz
-rw-r--r-- root/root      1430 2023-12-06 07:50 ./usr/share/man/man8/service.8.gz
-rw-r--r-- root/root      2560 2023-12-06 07:50 ./usr/share/man/man8/update-rc.d.8.gz
drwxr-xr-x root/root         0 2023-12-06 07:50 ./var/
drwxr-xr-x root/root         0 2023-12-06 07:50 ./var/lib/
drwxr-xr-x root/root         0 2023-12-06 07:50 ./var/lib/systemd/
```

Reply:
If I use the link you shared as reference, I should only have the `bins:` slice and remove `config:` and `var:`, correct?

Reply:
If they are listed by `dpkg -c` then I guess it's ok to have them since they are empty folfders anyway.